### DISCUSSION TITLE
msvcp140_1.dll was not found

### DISCUSSION THREAD

**Discussion by Mabru83** [opentrack/opentrack#1632] [Q&A] [answered]
Hello everyone,
after installing opentrack using the opentrack-2023.1.0-win32-setup, i get the error: the code execution cannot proceed because msvcp140_1.dll was not found

I'm on win 10 64 bit. (Am I missing something with the win 32 installer? This is the only one I was able to find...) Visual studio C++ 2017 was installed, i tried unistalling it and installing the 2015 version, but the problem is still there. 
What can I do? 
Thank you in advance.

**Comment by sthalik** [comment, marked as answer]
Unpack this where `opentrack.exe` is located.

[Microsoft.VC143.CRT.zip](https://github.com/opentrack/opentrack/files/11056888/Microsoft.VC143.CRT.zip)

**Reply by Mabru83** [reply]
Thank you! It solved the problem!
